src/components/message-center/style/message-center.scss

```diff
@@ -0,0 +1,5 @@
+
+
+[data-focus="snackbar-message-center"] {
+    
```

Comment:
Juste to allow a develop to know what he can overload. Do i remove it ? (we have done it on others components)